ops/testing.py

```diff
@@ -221,6 +225,22 @@ def set_can_connect(self, container: Union[str, model.Container], val: bool):
             container = self.model.unit.get_container(container)
         self._backend._set_can_connect(container._pebble, val)
 
+    def get_container_filesystem(
+        self, container: Union[str, model.Container]
+    ) -> "ContainerFilesystem":
+        """Return an interface for conducting filesystem operations on the specified container.
+
+        This method returns an object which provides an interface for filesystem-related
+        operations (push, pull, and so on) on the specified container. The operations available
+        through this interface are always permitted and unaffected by the
+        :meth:`.set_can_connect` method.
+        """
+        if isinstance(container, str):
+            container = self.model.unit.get_container(container)
+        return _TestingContainerFilesystemProxy(
+            container=container, backend=self._backend
+        )  # type: ignore
```

Comment:
Instead of a `type: ignore` comment, how about using `cast()`? Also, I'm not sure passing the args as keyword args adds much here.

```suggestion
        proxy = _TestingContainerFilesystemProxy(container, self._backend)
        return cast(ContainerFilesystem, proxy)
```

Reply:
Updated to use `cast` instead, thanks!